Jenkinsfile_CNP

```diff
@@ -1,7 +1,11 @@
 #!groovy
 
-List<LinkedHashMap<String, Object>> secrets = [
-  secret('microservicekey-send-letter-tests', 'TEST_SERVICE_SECRET')
+def secrets = [
+    's2s-${env}':
+        [
+            secret('jwt-key', 'JWT-KEY'),
```

Comment:
this isn't a valid env var, based on https://github.com/hmcts/service-auth-provider-app/blob/master/infrastructure/main.tf#L79
you probably want:
```suggestion
            secret('jwt-key', 'JWT_KEY'),
```

otherwise it'll be:
```suggestion
            secret('jwt-key', 'JWTKEY'),
```